### PR TITLE
add get gamut information

### PR DESCRIPTION
**Pull request by starkillerOG**
This Pull Request is needed for the "Improve Philips Hue color conversion" PR in HomeAssistant:
https://github.com/home-assistant/home-assistant/pull/19123
This adds the capability to get and parse the color gamut information that is supplied by the Hue bridge.
Tested this PR with HomeAssistant and everthing works just fine.

### CODE REVIEW

**Comment by starkillerOG** [comment]
@balloob as discussed in the ["Improve Philips Hue color conversion"](home-assistant/home-assistant#19123) PR in HomeAssistant, I am tagging you to expedite the review.
Can you review this PR and if it is good, can you merge and push to the next version of aiohue?

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 aiohttp
+attrs
```
Comment (balloob) [inline requirements.txt]: Let's not add a dependency for this. Use a namedtuple instead.

Comment (starkillerOG) [reply requirements.txt]: Alright I just switched to namedtuple instead of attrs

**Comment by starkillerOG** [comment]
@balloob I just switched to namedtupple instead of attrs.
Any more suggestions/inprovements, or can it be merged?

**Comment by balloob** [comment]
Released as aiohue 1.8.0

**Pull request merged**